core/src/main/java/org/mskcc/cbio/portal/servlet/PatientView.java

```diff
@@ -82,11 +82,36 @@ protected void processRequest(HttpServletRequest request, HttpServletResponse re
             throws ServletException, IOException {
         XDebug xdebug = new XDebug( request );
         request.setAttribute(QueryBuilder.XDEBUG_OBJECT, xdebug);
-                
-        request.setAttribute(QueryBuilder.HTML_TITLE, "Patient");
-        
-        RequestDispatcher dispatcher = getServletContext().getRequestDispatcher("/WEB-INF/jsp/patient_view/patient_view.jsp");
-        dispatcher.forward(request, response);
+
+        // Convert old URL parameters cancer_study_id and case_id/sample_id to new frontend
+        // hash style URL paramters #studyId=x&caseId=y
+        System.out.println(((HttpServletRequest)request).getRequestURL().toString());
+        String cancerStudyId = (String) request.getParameter("cancer_study_id");
+        System.out.println(cancerStudyId);
+        if (cancerStudyId != null) {
+            String hashUrl = "#/patient&studyId=" + cancerStudyId;
```

Comment:
as in:
http://www.cbioportal.org/case.do#/patient?studyId=ucec_tcga_pub&caseId=TCGA-BK-A0CC